### ISSUE TITLE
What if your IP address is block for no reason?

### ISSUE THREAD

**Issue by theartofweb** [MicrosoftDocs/microsoft-365-docs#7518]
On or around December 22, 2022 Microsoft appear to have black-listed the entire Linode network from sending emails.  As of January 27, 2022 several Outlook mail servers are still refusing to accept emails from Linode network addresses.  None of the suggested delisting procedures are of any use as they only result in "your IP address is not currently blocked".

More details: https://www.theregister.com/2022/01/17/linode_microsoft_email/

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 390d461b-7fce-d4ac-1c90-97b7b81dc0c2
* Version Independent ID: 9f8c7285-627f-13c6-1c7a-88fa7e14ed86
* Content: [Remove yourself from the blocked senders list - Office 365](https://docs.microsoft.com/en-gb/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis.md)
* Product: **m365-security**
* Technology: **mdo**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by theartofweb** [comment]
If I was a Microsoft user, and the only one affected, perhaps that would 
work.  Instead it is our clients' websites and businesses you are 
affecting.

Details:

 	https://www.theregister.com/2022/01/17/linode_microsoft_email/

Discussions:

 	https://docs.microsoft.com/en-us/answers/questions/674558/550-57511-access-denied-banned-sender.html
 	https://techcommunity.microsoft.com/t5/office-365/trying-to-find-help-for-550-5-7-511-access-denied-banned-sender/m-p/3044467?search-action-id=394163380403&search-result-uid=3044467
 	https://www.linode.com/community/questions/22287/550-57511-access-denied-banned-sender-office-365

As you can see, all 'support' avenues have already been attempted.

-- 
  Duncan Crombie                                    Chirp Internet S.R.L.
  ***@***.***>                  https://www.chirpinternet.eu

**Comment by msbemba** [comment]
@theartofweb Thank you for your response. I understand your situation, but from the docs team, we are not able to verify or troubleshoot this issue. This needs to go to Microsoft support whether it is a bug or an issue with a specific network.

Thanks
Sri

**Comment by theartofweb** [comment]
I would just like to point out that it's absolution #$%^ impossible to 
open a support ticket at 
https://support.serviceshub.microsoft.com/supportforbusiness for this 
issue when we are not using any particular Product.

So my point remains.  Your documentation is inadequate in that it doesn't 
provide for the situation where a Microsoft snafu has arbitrarily blocked 
a large range of ip addresses belonging to Lindoe for no reason.

https://www.theregister.com/2022/01/17/linode_microsoft_email/

-- 
  Duncan Crombie                                    Chirp Internet S.R.L.
  ***@***.***>                  https://www.chirpinternet.eu

**Comment by msbemba** [comment]
@theartofweb Thank you for the response. The documentation we have are the only steps available for an admin/user to remove from the blocked list. Anything outside of this is handled by the support team as there could be different reasons for the block. You can reach out to the support team via phone as well -https://docs.microsoft.com/en-us/microsoft-365/admin/get-help-support?view=o365-worldwide#phone-support

Thanks
Sri

**Comment by theartofweb** [comment]
Support. Won't. Talk. To. Us. Because. We. Are. Not. Customers.

https://techcommunity.microsoft.com/t5/office-365/trying-to-find-help-for-550-5-7-511-access-denied-banned-sender/m-p/3044467

-- 
  Duncan Crombie                                    Chirp Internet S.R.L.
  ***@***.***>                  https://www.chirpinternet.eu

**Comment by tiburd** [comment]
This is not a Docs issue. However, the Docs team is following up with the product group. IP de-listings are a long-standing issue that they are working on.

I'll update when I have more info.

**Comment by theartofweb** [comment]
Thank you.  We are exploring all channels to have this resolved by 
Microsoft.  Unfortunately, only the Docs team has responded in any way.

We understand that Linode has been working on this with the Microsoft 
'back end' team since the incident occurred on 22 December and has had 
some success in de-listing some ip ranges.

But at least two of our servers are still blocked from sending to many 
Outlook 365 servers - including some of our clients, their members and 
customers - and the documented de-listing process reports 'no block'.

Yours,
-- 
  Duncan Crombie                                    Chirp Internet S.R.L.
  ***@***.***>                  https://www.chirpinternet.eu

**Comment by tiburd** [comment]
Thanks again for your feedback about this. Because this isn't specifically a documentation issue, we've followed up with the product and support teams to help improve this situation. As a result, we've added additional information about the 5.7.511 error code and added specific links and email aliases to follow up in the event of an de-listing. This should result in a more expedient response from Microsoft.
